jobqueue/src/main/java/com/birbit/android/jobqueue/config/Configuration.java

```diff
@@ -358,10 +395,28 @@ public Builder consumerThreadPriority(int threadPriority) {
          *
          * @return The builder
          */
-        public Builder scheduler(Scheduler scheduler) {
+        @NonNull
+        public Builder scheduler(@NonNull Scheduler scheduler) {
             return scheduler(scheduler, true);
         }
 
+        /**
+         * Provide a factory class to create new worker instances for when JobManager needs them.<br />
+         * When a factory is installed, it becomes its responsability to configure
```

Comment:
instead of saying "meaningless", say JobManager uses the thread as is w/o changing its priority